go.mod

```diff
@@ -51,7 +51,7 @@ require (
 	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/cloudfoundryreceiver v0.75.0
 	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/collectdreceiver v0.75.0
 	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/filelogreceiver v0.75.0
-	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/fluentforwardreceiver v0.75.1-0.20230405204547-23c6e57b5d78
+	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/fluentforwardreceiver v0.75.0
```

Comment:
this change should not be reverted, we need it to pick up the fluentforwardreceiver changes

Reply:
Oh jeeze, thank you.  This is why you don't blindly follow a runbook.  Forced pushed change/revert

Reply:
There's also new automation to take care of the otel dep bumps for us: https://github.com/signalfx/splunk-otel-collector/pull/2920